docs/service-bundle.md

```diff
@@ -127,12 +129,18 @@ Example of JSON document passed to __bind__:
   provision action for this service instance. Details are below in the Output
   section.
 
+#### Last Request User
+
+The requesting username of the [actions](#actions) _provision_, _deprovision_, _bind_, _unbind_, and _update_ is available in the `_apb_last_requesting_user` parameter. The parameter is set to the `UID` if the requesting `username` is empty (e.g. auto escalation). Also, the user info of the latest request will always overwrite the existing value.
```

Comment:
I think we should note that this could be completly empty as it is not a requirement to send the requesting user.

Sorry @johnkim76 one more change?

Reply:
thanks, will update